### PR TITLE
Mock screen for editing surface tokens with tabs and without content for each tab

### PR DESCRIPTION
**Pull request by LinconOssa**
The implementation of tabs to Surfaces is done following the aesthetics of figma

### CODE REVIEW

File: src/pages/people/outlets/surfaces/interface.tsx [hunk 1]
```diff
@@ -86,12 +82,22 @@ export function SurfacesUI(props: UsersUIProps) {
               navigatePage="/people"
             />
           </Stack>
-          <StyledContainer>
-            <Stack gap="32px" direction="column"></Stack>
-          </StyledContainer>
         </Stack>
-        {renderMessage(message, handleCloseMessage)}
+
+        <StyledContainer>
```
Comment (carin91) [inline src/pages/people/outlets/surfaces/interface.tsx:87]: Why do you need all of these containers? StyledContainer, Stack and StyledTabsContainer, i think it is too much. The Tabs component is not enough?

**Review by carin91 — DISMISSED** [review]

File: src/pages/people/outlets/surfaces/interface.tsx [hunk 2]
```diff
@@ -1,25 +1,22 @@
...
+  StyledContainer,
+  StyledTabsContainer,
+} from "./styles";
 import { IUsersMessage } from "@src/pages/privileges/outlets/users/types/users.types";
-import { privilegeUserTabsConfig } from "@src/pages/privileges/outlets/users/config/usersTabs.config";
-import { menuInvitationLinks } from "@src/pages/privileges/outlets/users/config/menuInvitation.config";
-import { InvitationsTab } from "@src/pages/privileges/outlets/users/tabs/invitations";
-import { UsersTab } from "@src/pages/privileges/outlets/users/tabs/users";
 import { peopleOptionsConfig } from "../options/config/people.config";
+import { colorTabsConfig } from "./config/colorTabs.config";
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/pages/people/outlets/surfaces/interface.tsx:20]: Resolved at the meeting

File: src/pages/people/outlets/surfaces/interface.tsx [hunk 3]
```diff
@@ -57,25 +55,23 @@ interface UsersUIProps {
 
 export function SurfacesUI(props: UsersUIProps) {
   const {
-    isSelected,
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/pages/people/outlets/surfaces/interface.tsx:60]: I agreed with my partner Cesar to delete it

File: src/pages/people/outlets/surfaces/interface.tsx (hunk 1 above)
Comment (LinconOssa) [reply src/pages/people/outlets/surfaces/interface.tsx:87]: According to what we agreed with Cesar and with you in the meeting, it would be fine.

**Review by cmarin001 — APPROVED** [review]
meet the requirements

**Pull request merged**